cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,36 @@
 /// <reference types='cypress' />
-
 describe('Book Store app', () => {
-  before(() => {
-    
+  before(() => { });
+
+
+  it('login', () => {
+    cy.login();
+    cy.visit('/profile');
+    cy.get('#userName-value').should('contain', 'dmukhailo123');
+    cy.url().should('include', '/profile');
   });
 
-  it('', () => {
-    
+  it('add book', () => {
+    cy.viewport(750, 600);
+    cy.login();
+    cy.visit('/profile');
+    cy.contains('.btn', 'Book Store').click();
+    cy.get('[placeholder="Type to search"]').click().type('Speaking JavaScript');
+    cy.contains('O\'Reilly Media');
+    cy.contains('.action-buttons', 'Speaking JavaScrip').click();
```

Comment:
There's a typo in the book title, it should be 'Speaking JavaScript' instead of 'Speaking JavaScrip'. Please correct it to avoid potential issues in the test.